code/game/dna/genes/vg_disabilities.dm

```diff
@@ -69,3 +69,46 @@
 
 	OnSay(var/mob/M, var/datum/speech/speech)
 		speech.message_classes.Add("sans") // SPEECH 2.0!!!1
+
+/datum/dna/gene/disability/veganism
+	name = "Veganism"
+	desc = "Causes the digestive system to completely reject all animal products, from meat to dairy."
+	activation_message = "You feel vegan."
+	deactivation_message = "You're back on top of the food chain."
+
+	mutation = M_VEGAN
+
+	var/static/list/bad_reagents = list(
```

Comment:
Can you really not make this a variable at the reagent level

Reply:
Done, now a global variable